go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/kyleconroy/sqlc v1.16.0
 	github.com/kylelemons/godebug v1.1.0
 	github.com/lib/pq v1.10.7
+	github.com/markbates/pkger v0.15.1
```

Comment:
`pkger` has been removed, this may be because of the `pkged.go` file still be locally?

Reply:
Ah good catch, thank you, that was exactly the issue. Addressed now!